pyproject.toml

```diff
@@ -106,13 +106,14 @@ exclude = [
     "httpcore/_sync",
     "tests/_sync",
 ]
-line-length = 120
+line-length = 88
 select = [
     "E",
     "F",
     "W",
     "I"
 ]
 
+ignore = ["E501"]
```

Comment:
I suggest use [tool.ruff.pycodestyle.max-line-length](https://docs.astral.sh/ruff/settings/#pycodestyle-max-line-length) with value of 120 instead of ignore it.